students/grzegorz_kowalik/lesson_01_basics/hello_harry.py

```diff
@@ -0,0 +1,2 @@
+print("What's your name?")
+print("Hello, " + input() + "!")
```

Comment:
można to jeszcze skrócić do jednej linii przenosząc pierwszy komunikat do funkcji input().
Gdybyś użył input() w print() w python2.x to byś dostał błąd NameError.